src/camera/camera_pinhole.h

```diff
@@ -32,119 +32,51 @@
   }
-  
+
   template <typename Derived>
   inline Eigen::Vector2f UnprojectFromImageCoordinates(const Eigen::MatrixBase<Derived>& pixel_position) const {
     return Undistort(Eigen::Vector2f(fx_inv() * pixel_position.x() + cx_inv(), fy_inv() * pixel_position.y() + cy_inv()));
   }
-  
+
   template <typename Derived>
   inline Eigen::Vector2f Undistort(const Eigen::MatrixBase<Derived>& normalized_point) const {
     return normalized_point;
   }
-  
-  // Returns the derivatives of the normalized projected coordinates with
-  // respect to the 3D change of the input point.
-  template <typename Derived>
-  inline void ProjectionToNormalizedTextureCoordinatesDerivative(
-      const Eigen::MatrixBase<Derived>& point, Eigen::Vector3f* deriv_x, Eigen::Vector3f* deriv_y) const {
-    const Eigen::Vector2f normalized_point = Eigen::Vector2f(point.x() / point.z(), point.y() / point.z());
-    const Eigen::Vector4f distortion_deriv = DistortionDerivative(normalized_point);
-    const Eigen::Vector4f projection_deriv =
-        Eigen::Vector4f(nfx() * distortion_deriv.x(), nfx() * distortion_deriv.y(),
-                        nfy() * distortion_deriv.z(), nfy() * distortion_deriv.w());
-    *deriv_x = Eigen::Vector3f(
-        projection_deriv.x() / point.z(), projection_deriv.y() / point.z(),
-        -1.0f * (projection_deriv.x() * point.x() + projection_deriv.y() * point.y()) /
-            (point.z() * point.z()));
-    *deriv_y = Eigen::Vector3f(
-        projection_deriv.z() / point.z(), projection_deriv.w() / point.z(),
-        -1.0f * (projection_deriv.z() * point.x() + projection_deriv.w() * point.y()) /
-            (point.z() * point.z()));
-  }
-  
-  // Returns the derivatives of the image coordinates with respect to the 3D
-  // change of the input point.
-  template <typename Derived>
-  inline void ProjectionToImageCoordinatesDerivative(
-      const Eigen::MatrixBase<Derived>& point, Eigen::Vector3f* deriv_x, Eigen::Vector3f* deriv_y) const {
-    const Eigen::Vector2f normalized_point = Eigen::Vector2f(point.x() / point.z(), point.y() / point.z());
-    const Eigen::Vector4f distortion_deriv = DistortionDerivative(normalized_point);
-    const Eigen::Vector4f projection_deriv =
-        Eigen::Vector4f(fx() * distortion_deriv.x(), fx() * distortion_deriv.y(),
-                        fy() * distortion_deriv.z(), fy() * distortion_deriv.w());
-    *deriv_x = Eigen::Vector3f(
-        projection_deriv.x() / point.z(), projection_deriv.y() / point.z(),
-        -1.0f * (projection_deriv.x() * point.x() + projection_deriv.y() * point.y()) /
-            (point.z() * point.z()));
-    *deriv_y = Eigen::Vector3f(
-        projection_deriv.z() / point.z(), projection_deriv.w() / point.z(),
-        -1.0f * (projection_deriv.z() * point.x() + projection_deriv.w() * point.y()) /
-            (point.z() * point.z()));
-  }
-  
+
   // Returns the derivatives of the image coordinates with respect to the
```

Comment:
It seems this comment is not appropriate for the function anymore. Right now the function does nothing.